turbo.json

```diff
@@ -14,10 +14,11 @@
       "cache": false
     },
     "test": {
-      "outputs": []
-    },
-    "test:templates": {
-      "outputs": []
+      "outputs": [],
+      "dependsOn": [
+        "$RUNNER_OS",
+        "$NODE_VERSION"
+      ]
```

Comment:
We needed to exposes these variables to `dependsOn` so that tests are not hashed only on the file contents. This means test runs will be hashed on the files contents *and* the current OS and version of Node.